app/routes/endemics/dev-sign-in.js

```diff
@@ -56,30 +57,40 @@ const postHandler = {
       request.logger.setBindings({ sbi })
       const [personSummary, organisationSummary] = await createDevDetails(sbi)
 
-      setEndemicsClaim(
-        request,
-        sessionKeys.endemicsClaim.organisation,
-        {
-          sbi: organisationSummary.organisation.sbi?.toString(),
-          farmerName: getPersonName(personSummary),
-          name: organisationSummary.organisation.name,
-          email: personSummary.email ? personSummary.email : organisationSummary.organisation.email,
-          orgEmail: organisationSummary.organisation.email,
-          address: getOrganisationAddress(organisationSummary.organisation.address),
-          crn: personSummary.customerReferenceNumber,
-          frn: organisationSummary.organisation.businessReference
-        }
-      )
+      try {
+        const latestApplication = await getLatestApplicationForSbi(
+          organisationSummary.organisation.sbi?.toString(),
+          organisationSummary.organisation.name
+        )
+
+        setEndemicsClaim(
+          request,
+          sessionKeys.endemicsClaim.organisation,
+          {
+            sbi: organisationSummary.organisation.sbi?.toString(),
```

Comment:
I know this is from my original, but might as well simplify any ref to sbi as it's in the params,and must be present